src/test/java/SetTest.java

```diff
@@ -0,0 +1,76 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.HashSet;
+import java.util.Set;
+
+
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    /**
+     * 요구사항 1
+     * Set의 size() 메소드를 활용해 Set의 크기를 확인하는 학습테스트를 구현한다.
+     */
+    @Test
+    @DisplayName("요구사항1")
+    void setTestNo1(){
+        int numbersSize = numbers.size();
+
+        // numbers의 크기가 3인지 확인
+        assertThat(numbersSize).isEqualTo(3);
```

Comment:
assertJ에선, size를 검증하기 위해 hasSize()라는 메서드를 제공합니다. 아래와 같이 작성할수 있습니다. 한번 참고해보세요~

```suggestion
        assertThat(numbers).hasSize(3);
```

Reply:
좋은 팁 감사합니다! step2 에 반영하도록 하겠습니다 👍